webapp/src/components/EstateDetailPage/EstateDetailPage.js

```diff
@@ -41,27 +41,26 @@ export default class EstateDetailPage extends React.PureComponent {
     return (
       <Estate assetId={assetId} x={x} y={y}>
         {(estate, isOwner, wallet) =>
-          estate.data.parcels.length ? (
-            isNewAsset(estate) || isEditing ? (
-              <EditEstate
-                estate={estate}
-                isCreation={isNewAsset(estate)}
-                isOwner={isOwner}
-                wallet={wallet}
-                onViewAssetClick={onViewAssetClick}
-                isSelecting={isNewAsset(estate) || isSelecting}
-              />
-            ) : (
-              <EstateDetail
-                allParcels={allParcels}
-                estate={estate}
-                isOwner={isOwner}
-                onViewAssetClick={onViewAssetClick}
-                onEditParcels={onEditParcels}
-                onEditMetadata={onEditMetadata}
-              />
-            )
-          ) : null
+          estate.data.parcels.length > 0 &&
```

Comment:
I think we can remove `estate.data.parcels.length > 0` I added it to not render anything in case the estate is empty. As we have that check [here](https://github.com/decentraland/marketplace/pull/389/files#diff-dc94e71ea3fc145084f572f535ae7185R50) maybe it is obsolete to have it here too